src/main/java/blusunrize/immersiveengineering/common/blocks/IEMultiblockBlock.java

```diff
@@ -63,8 +67,11 @@ public void onReplaced(BlockState state, World world, BlockPos pos, BlockState n
 	}
 
 	@Override
-	public void getDrops(BlockState state, NonNullList<ItemStack> drops, World world, BlockPos pos, int fortune)
+	@SuppressWarnings("deprecation")
+	public List<ItemStack> getDrops(BlockState state, LootContext.Builder builder)
 	{
+		//TODO review: Presuming the drop list has to be empty, as items are manually spawned in `onReplaced()` above.
```

Comment:
Yes. It might be possible to move spawning here with the loot context, but I don't think it would be useful in this case.

Reply:
Removed the comment accordingly.